### ISSUE TITLE
Binaries are not assembled

### ISSUE THREAD

**Issue by martinz01** [entropia/tip-toi-reveng#178]
I have exported an existing gme file with media and binaries. After "assemble" them, the binaries will not be put into gme file.

**Comment by nomeata** [comment]
Yes, that’s a known problem. Until we know how the binaries refer to the media files, we cannot do that. But it’s a fun project for anyone who wants to reverse-engineer this part of the GME format!
